### DISCUSSION TITLE
HTTP Error 500.0 - Internal Server Error

### DISCUSSION THREAD

**Discussion by landaisgu** [episerver/Foundation#725] [Q&A]
After running the setup.cmd file, my website thrown out the HTTP Error 500.0 - Internal Server Error, then I turned on Web.config to the debug mode by the code below
![image](https://user-images.githubusercontent.com/5188174/119996361-e03fbc00-bff8-11eb-9db1-3ee52df1b190.png)
and after that,  my website thrown out a error as the below image. Can you help me?

![The-remote-server-returned-an-error-403-Forbidden-](https://user-images.githubusercontent.com/5188174/119986835-8b974380-bfee-11eb-9d68-e9ae5185bc1e.png)

**Comment by michal-mitas** [comment]
It's a bit late but I hope this helps someone in the future. Error indicates 403 response from EpiServer Find. You need to create a demo search index on https://find.episerver.com/ and put it in a Web.config. Replace the line `<episerver.find serviceUrl="https://es-us-api01.episerver.com/9IKGqgMZaTD9KP4Op3ygsVB6JeJzR0N6" defaultIndex="episerverab_index55794" />` with a value you retrieve from find.episerver.com
